osu.ElasticIndexer/Commands/PushFileCommand.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.ComponentModel.DataAnnotations;
+using System.IO;
+using System.Threading;
+using McMaster.Extensions.CommandLineUtils;
+
+namespace osu.ElasticIndexer.Commands
+{
+    [Command("push-file", Description = "Push contents of a file to the queue for testing.")]
+    public class PushFileCommand : ProcessorCommandBase
+    {
+        [Argument(0)]
+        [Required]
+        public string Filename { get; set; } = string.Empty;
+
+        public int OnExecute(CancellationToken cancellationToken)
+        {
+            if (string.IsNullOrEmpty(AppSettings.Schema))
+                throw new MissingSchemaException();
+
+            var value = File.ReadAllText(Filename);
+
+            var redis = new Redis();
+            var database = redis.Connection.GetDatabase();
+            var queueName = $"osu-queue:score-index-{AppSettings.Schema}";
+
+            database.ListLeftPush(queueName, value);
```

Comment:
Any reason for doing it like this rather than using the existing queue method? I'd much prefer the latter.

Reply:
But you can just `new ScoreItem { id = ... }` no?

Reply:
Pushing directly to the key allows pushing things like `{ "ScoreId": 1, "not_a_real_field": "aa" }` and also intentionally omit properties that didn't exist in other versions, instead of it being serialized into to type of the current version, which isn't as useful when testing for payloads from external sources. 

`processor.PushToQueue(new ScoreItem())` is always going to push `"{\"Action\":null,\"ScoreId\":null,\"Score\":null,\"TotalRetries\":0,\"Tags\":null}"` to the queue; a different client isn't necessarily going to be sending all the fields.

Reply:
I guess it's fine to leave this, since this command probably won't be used in production. Still a bit weird though. I can imagine this breaking if we change the queue name or system and forget to update this. *shrug*